docs/TutorialReact.md

```diff
@@ -225,6 +225,10 @@ mocked module fails these checks. Your options are:
     types and will happily accept e.g. `SomeComponent`. You could check
     snapshots using the test renderer, and check component behavior separately
     using Enzyme.
+4.  Disable warnings all together (should be done in your jest setup file):
+    ```js
+    jest.mock('fbjs/lib/warning', () => require('fbjs/lib/emptyFunction'));
+    ```
```

Comment:
My gut is that disabling all of the warnings is unwise, or at least something that shouldn't be officially recommended

Reply:
I hear you but there is a use case with react-native + react 16 where this is the the only option as you can't use react-test-renderer. The problem here is that we are rendering react-native tags into the DOM and these warnings are irrelevant to react-native. There's actually a better solution - https://github.com/Root-App/react-native-mock-render/issues/36#issuecomment-379432890

Not sure if it makes sense to document it here. I'll be happy to update this PR if you think it is or just close it.

Reply:
Ah, makes sense. What do you think about adding a word of caution?

Reply:
@rickhanlonii sorry for the delay. Added :)